src/test/java/harness/utils/WorkspaceUtils.java

```diff
@@ -11,6 +11,10 @@
 /** Utilities for working with workspaces in CLI tests. */
 public class WorkspaceUtils {
 
+  public static String createUserFacingId() {
+    return "a-" + UUID.randomUUID().toString();
```

Comment:
So userFacingId must start with letter ([see here](https://docs.google.com/document/d/19urXR2VMmX2K0Z-_m1NibVnnFPe6gM1ezrnCfWW3044/edit#heading=h.7e8e1y516i10)).

WorkspaceManager handles workspace creation. If someone calls WorkspaceService.createWorkspace and doesn't pass userFacingId, WorkspaceManager generates one with `a-` + uuid ([code](https://github.com/DataBiosphere/terra-workspace-manager/blob/db15c6c74726f33e7acecd0ad67eb9812729f82f/service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java#L403)). `a` is because userFacingId must start with letter. `-` to make it easier to tell userFacingId apart from guid. (guid looks like d4061a38-a221-497e-8f01-4c61e5bbe275).

In this test, we copy this convention.

Reply:
thanks for the referring. good to learn.